mimetic/rfc822/mailbox.cxx

```diff
@@ -61,7 +61,9 @@ void Mailbox::set(const string& input)
     // the string::erase or/and we cannot cache begin() end()
 
     int t = input.length() -1;
-    if(input[t] == '>')
+    while(input[t] == ' ')
```

Comment:
`while(t && input[t] == ' ')` looks better to avoid having t less than zero for strings having just whitespaces.

Reply:
Thanks. I will modify and make new pull request.